tests/cli/commands/test_dag_command.py

```diff
@@ -161,7 +161,7 @@ def test_backfill(self, mock_run):
             )
 
         output = stdout.getvalue()
-        assert f"Dry run of DAG example_bash_operator on {DEFAULT_DATE.isoformat()}\n" in output
+        assert f"Dry run of DAG example_bash_operator on {DEFAULT_DATE.isoformat(sep=' ')}\n" in output
```

Comment:
Because `penudlum.DateTime` now have less overwrite on `datetime.datetime` it change the behaviour of convert to string

**Pendulum 2**

```python
>>> import pendulum
>>> str(pendulum.now())
'2023-12-18T23:14:57.788852+04:00'
``` 

**Pendulum 3**

```python
>>> import pendulum
>>> str(pendulum.now())
'2023-12-18 23:14:57.788852+04:00'
```